SQL 26th sep/26sep'24.txt

```diff
@@ -0,0 +1,78 @@
+1.) 
+create database employee2;
+use employee2;
+CREATE TABLE Employees (  EmployeeID INT PRIMARY KEY,FirstName VARCHAR(50), LastName VARCHAR(50),  Department VARCHAR(50),Salary DECIMAL(10, 2),HireDate DATE);
+select * from Employees;
+insert into Employees values (1,'aaa','Abin','Marketing',15000, '2002-07-20');
+insert into Employees values (2,'bbb','beni','sales ', 60000, '2020-08-01');
+insert into Employees values (3,'ccc','Cyril','IT',87000, '2019');
+insert into Employees values (4,'John','daniel','marketing ', 60000, '2019-10-09');
+insert into Employees values (5,'John','Saniel','marketing ', 75000, '2007-10-09');
+insert into Employees values (6,'Krish','Saniel','IT', 50000, '2010-10-09');
+
+create table students(name nvarchar,rollno int PRIMARY KEY,name nvarchar(30));
```

Comment:
You have not mentioned the length of the column "name" ? why ? also there are 2 columns with same name , will this work ?

Reply:
sorry, a typing mistake while doing it to notepad trying to make  its print order better ,its only one time , its actually like:
 create table students(rollno int PRIMARY KEY,name nvarchar(30));